### ISSUE TITLE
No output during pod installation - exits early

### ISSUE THREAD

**Issue by andrewhavens** [HipByte/motion-cocoapods#129]
My Gemfile.lock shows that I am using motion-cocoapods 1.7.4. When I run `bundle exec rake pod:install` I see the output of "updating spec repo", then I see the message that CocoaPods 0.39.0 is available. Then it exits. The pods are not being installed. I launch my app and I get method missing errors on classes that should exist from my installed pods.

**Comment by MarkVillacampa** [comment]
Hello, can you provide a sample app that reprocuces this error?

**Comment by andrewhavens** [comment]
Actually, I think the problem is that there is no output when all of the pods are installed. I made a change to my list of pods and rake `bundle exec rake pod:install` again and was able to see the pods install successfully. When I ran it again, the output was blank. However, it seems that "re-installing" fixed my method missing errors. I think the solution to this problem is to display a message that all of the pods have been installed, or that the dependencies are up to date, similar to running `bundle check`.
